src/GameState/Marble.tsx

```diff
@@ -4,28 +4,29 @@ import React from 'react';
 const MARBLE_COLOR_RADIAL = ['warmGray.300', 'black', 'white', 'red.700'];
 
 export const Marble = (props: any) => {
-
-
   const size = props.size;
-  const child = <Circle size={size} m={0} p={0} bgColor={getMarbleColor(props.value)} />;
+  const child = (
+    <Circle size={size} m={0} p={0} bgColor={getMarbleColor(props.value)} />
+  );
 
   const marbleClick = async () => {
-    await props.setMarbleClicked({ x: props.cellIndex, y: props.rowIndex, value: props.value, isExit: false })
-    props.setMarbleClickedCoordinates({ x: props.cellIndex, y: props.rowIndex })
-  }
+    await props.setMarbleClicked({
```

Comment:
It's difficult to understand why this method is async and why the next one is not. The naming is not good IMO.

Reply:
There was no reason, I removed it